src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -741,6 +742,10 @@ impl<'a> TypeChecker<'a> {
                 {
                     if let Some(udf) = self.resolve_udf(*span, func_name, args).await? {
                         return Ok(udf);
+                    } else if let Some(async_function) =
```

Comment:
Better to add a const array like `GENERAL_WINDOW_FUNCTIONS` to check whether the function is a async function. This can help us to check the function names and avoid the udf function having the same name.